docs/src/increase-observability/metrics/_index.md

```diff
@@ -92,3 +92,14 @@ The interval between measurements then changes based on the range you choose.
 | < 2 hours       | 1 minute                  |
 | 2 to 5 hours    | 5 minutes                 |
 | 5 to 8/24 hours | 20 minutes                |
+
+## Deployments
+
+Sometimes deployment activities (pushes, merges, syncs) can influence the underlying infrastructure.
+You don't want to confuse a spike caused by a successful deploy with an issue that needs your attention.
+
+To see how deployment activity influences the infrastructure, turn on **Deployments** on your metrics.
+
+Each deployment activity appears as a line on each graph corresponding to the time the activity finished.
```

Comment:
❓ 
Should we include the deployments "eye toggle" and maybe some part of the graph with a line as screenshots for that section? Or update the other screenshots?

Reply:
I don't think we're after a full documentation of the UI. If you just mean the Deployments toggle (not sure what the "eye" part means), I don't think it's fully necessary.

Recreating the other screenshots might be good, it's just a hassle to get the data similar or similarly interesting. I thought it might not be worth the effort, but I might be able to show something not too much less interesting.

Reply:
Yes, I meant the deployment toggle :)
The new images are great 😍